schema.graphql

```diff
@@ -495,11 +495,17 @@ type Bid @entity {
   bidder: Account!
 
   "The value of the bid"
-  value: BigInt!
+  value: BigInt
```

Comment:
nit: I think still makes sense to calculate Bid Value for RAM minter - see solidity function in RAMLib and could duplicate that calc in the subgraph handler.

Reply:
Good call, added this helper 🙏